packages/core/tests/span.test.ts

```diff
@@ -354,3 +359,61 @@ describe('Span', () => {
     })
   })
 })
+
+describe('SpanContext', () => {
+  describe('SpanContext.isValid()', () => {
+    it('returns false if the span has been ended', () => {
+      const delivery = new InMemoryDelivery()
+      const clock = new IncrementingClock('1970-01-01T00:00:00Z')
+      const client = createTestClient({ deliveryFactory: () => delivery, clock })
+      client.start({ apiKey: VALID_API_KEY })
+
+      const span = client.startSpan('test span')
+      expect(span.isValid()).toEqual(true)
+
+      span.end()
+      expect(span.isValid()).toEqual(false)
+    })
+  })
+  describe('spanContextEquals()', () => {
+    it.each([
+      {
+        s1: undefined,
+        s2: undefined,
```

Comment:
Can these be more descriptive? i.e. `span1` & `span2`, `context1` & `context2` or whatever